tools/appveyor.psm1

```diff
@@ -85,6 +85,13 @@ Function Test-DailyBuild
     {
         return $true
     }
+    
+    # if [Feature] is in the commit message,
+    # Run Daily tests
+    if($env:APPVEYOR_REPO_COMMIT_MESSAGE -match '(\[(?i)feature\]){1}')
```

Comment:
i think this need only be:
`-match '\[feature\]'`
you don't seem to be doing anything with the match, also our `-match` is case insensitive by default